x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -4,13 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { FtrProviderContext } from '../../ftr_provider_context';
+import {
+  skipIfNoRegistry,
+  isRegistryEnabled,
+} from '../../../security_solution_endpoint_api_int/registryry';
+
+export default function (providerContext: FtrProviderContext) {
+  const { loadTestFile, getService } = providerContext;
 
-export default function ({ loadTestFile, getService }: FtrProviderContext) {
   describe('endpoint', function () {
     this.tags('ciGroup7');
     const ingestManager = getService('ingestManager');
+    const log = getService('log');
+
+    skipIfNoRegistry(log);
     before(async () => {
-      await ingestManager.setup();
+      if (isRegistryEnabled()) {
+        await ingestManager.setup();
+      } else {
+        log.warning('skipping ingest setup because the registry location was not defined');
```

Comment:
But the test will fail if this is not enabled right? Is that what the skip question above is about?

Reply:
Yeah, currently the tests will be skipped if `INGEST_MANAGEMENT_PACKAGE_REGISTRY_PORT` and `PACKAGE_REGISTRY_OVERRIDE` are not defined. So they won't fail.